test/unit/source/deinitialization.d

```diff
@@ -0,0 +1,150 @@
+// global.deinitialize
+unittest
```

Comment:
I thought you wanted to add the `unittest` directly in the DMD source code?

Reply:
No (already replied elsewhere).